package.json

```diff
@@ -129,6 +131,12 @@
 			"\\.pegjs$": "<rootDir>/test/unit/pegjs-transform.js"
 		}
 	},
+	"lint-staged": {
+		"*.js": [
+			"eslint --fix",
+			"git add"
```

Comment:
I've found that this conflicts with patch/partial commits, at least those made via the [GitHub Desktop client](https://desktop.github.com/), presumably because `git add` will add all changes from the tested files, even though I'd already made only a partial selection to commit.

Reply:
@aduth Hmmm, I _think_ it relates to this lint-staged issue:
• https://github.com/okonet/lint-staged/issues/62 
There's an active WIP PR, but it's doesn't look like landing in the short term
• https://github.com/okonet/lint-staged/pull/75

Maybe we should drop this and revert until a solution for the above lands?